Sources/AppStoreConnectCLI/Commands/TestFlight/BetaGroups/Sync/SyncBetaGroupsCommand.swift

```diff
@@ -0,0 +1,17 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import ArgumentParser
+
+struct SyncBetaGroupsCommand: ParsableCommand {
+    static var configuration = CommandConfiguration(
+        commandName: "sync",
+        abstract: """
+        Sync information about beta groups with provided configuration file.
+        """,
+        subcommands: [
+            PullBetaGroupsCommand.self,
+            PushBetaGroupsCommand.self,
+        ],
+        defaultSubcommand: PullBetaGroupsCommand.self
```

Comment:
I don't think we should have a default command.
```suggestion
        ]
```